amivapi/validation.py

```diff
@@ -121,3 +122,22 @@ def _validate_filetype(self, filetype, field, value):
         if not(t in filetype):
             self._error(field, "filetype '%s' not supported, has to be in: "
                         "%s" % (t, filetype))
+
+    def _validate_session_younger_than(self, threshold_time, field, value):
```

Comment:
As the units of ```threshold_time``` are very important here, I would recommend including it in the parameter name or even better in the function name. How about ```_validate_session_younger_than_s```?

Reply:
I would leave the function name as is (in the end, you can pass whatever you want anyway), but defintitely include it in the validators docstring.

In particular, I would not use an integer argument at all, since it really is ambiguous when it comes to time units. I would use a `datetime.timedelta`, since it is
- more descriptive
- more flexible

e.g. you could use it like:
```
from datetime import timedelta
(...)
'session_younger_then': timedelta(seconds=1)
```

What do you think? Afaik it should work without any changes, since we are using differences of time objects anyway -- they can be compared to timedeltas by design

Reply:
yes good idea. Definitively go for timedelta.